packages/block-library/src/table/edit.js

```diff
@@ -590,7 +591,7 @@ export class TableEdit extends Component {
 					<ToolbarGroup>
 						<DropdownMenu
 							hasArrowIndicator
-							icon="editor-table"
+							icon={ tableEdit }
```

Comment:
I'm assuming table is used as a prefix here, so as to group related table icons together, is that correct? Or could we just use "table" instead?

Reply:
yes, that's the idea, I believe that icon is different from the "table" one (used for the block's icon).

Reply:
Oh this one?

![Screenshot 2020-02-17 at 11 08 21](https://user-images.githubusercontent.com/1204802/74643943-d385b480-5175-11ea-9816-d811df08ae9d.png)

Yeah this is an opportunity to unify those to be the same. Perhaps name this "table", and we just don't port the block icon?

Reply:
<img width="1128" alt="Capture d’écran 2020-02-17 à 9 16 56 AM" src="https://user-images.githubusercontent.com/272444/74644324-69214400-5176-11ea-91f6-a99d59515fb6.png">

In this toolbar, they are both shown at the same time, I believe it's a bit weird if we use the same icon.

Reply:
Yes, good point. But that's an argument for a better block icon perhaps suffixed "-block".

Reply:
Looks like there's still both icons in https://www.figma.com/file/fnyj380i05vGzuuH60frLQ/G2-Design?node-id=1178%3A0

Reply:
Yep but I'm not sure we should put block icons into the library, yet, or ever. We might just keep those as inline SVG data. But we do need a table icon 😅

Reply:
What's the reasoning for this? I believe we should put all icons there personally, it's easier to find and update without impact on bundle size or anything.

Reply:
The block type icons are not meant to be used through the `wordpress/icons` package, at least not initially. (e.g. whenever we use them in code, it should be coming from `blockType.icon` and not `icon={ blockName }`.)

The table example needs a better icon representation for the dropdown, or a text button. Even then, it should currently be slightly different:

![image](https://user-images.githubusercontent.com/548849/74647515-15b1f480-517c-11ea-8adc-9d484f2e6449.png)

vs

![image](https://user-images.githubusercontent.com/548849/74647539-22364d00-517c-11ea-9af0-bdca50dadd86.png)

Reply:
We could have a dedicated folder in `@wordpress/icons` for blocks. Since it's a npm package that is bundled, I see only benefits for putting the icons there, there's no WP APIs involved.

Noting that they are already there.

Reply:
My point is more towards taking no shortcuts around displaying a block icon through the icon package instead of the block API.

Reply:
Ok, well now the icons are already there. Let's move on with this PR which puts us closer to deprecating the Dashicon component. I'll also work on a storybook page to show the icons. I think it could help us make better decisions of what to keep,  rename and remove from @wordpress/icons

Reply:
Yep, and in this case it seems like the path forward is to just ignore the table block icon for now, and take on the name `table` for the icon used for editing the table. Does that sound okay?

Reply:
Not really, since the table icon is already in the package. It seems weird to make an exception for one block for now.

Reply:
Oooh, I see. 

I think we should consider renaming the block icon to `block-` then, because the basic table icon is likely to be used in a variety of places that isn't new blocks, and we'd rather they use the table icon than the table block icon for that.

Reply:
ok

Reply:
But is there going to be a "table icon"? I think we should end up removing it.

Reply:
I suppose we can replace this with something else:

![Screenshot 2020-02-17 at 15 33 18](https://user-images.githubusercontent.com/1204802/74662774-db0b8480-519a-11ea-9018-96ff4caf9ac4.png)

Could be a label or a more generic icon. But just from Docs/Spreadsheet precedence, the table icon seems okay there.